spec/support/shared_contexts/api/docs/api_update_endpoint_documentation_support.rb

```diff
@@ -26,6 +26,21 @@
       response "200", "The updated #{options[:resource_description]}" do
         schema({ "$ref": options[:response_schema_ref] })
 
+        after do |example|
+          if defined?(response_example)
```

Comment:
I wonder if we should stick this in a helper as we're going to pull it into most/all the shared examples at some point I reckon